localization/localization_util/include/localization_util/util_func.hpp

```diff
@@ -29,13 +29,20 @@
 #include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>
 #endif
 
+#include <Eigen/Dense>
+
 #include <algorithm>
 #include <cmath>
 #include <deque>
 #include <random>
 #include <string>
 #include <vector>
 
+std::vector<Eigen::Vector2d> create_initial_pose_offset_model(
```

Comment:
From my personal point of view, `localization_util` should have functions that are likely to be used in other packages.

It seems that `create_initial_pose_offset_model` and `calc_eigenvector_angle` are only used in `ndt_scan_matcher`, and they appear to be called only once, so I think it might not be a bad idea to write them directly at the point of use. 

This is not an absolute necessity for a change, and I welcome any opposing views or comments.

Reply:
I'm sorry. I asked Aoki-san to do so...

Reply:
OK, no problem.
Let's leave it as it is.

Reply:
Isn't `calc_eigenvector_angle` general purpose? (By the way, the function name is not so good.)

I do not think `create_initial_pose_offset_model` needs to be included in the localizaton_util. Because it is very limited usage.

Reply:
Two functions, create_initial_pose_offset_model and calc_eigenvector_angle are limited usage.
Can I put these functions in ndt_scan_matcher_core.cpp as **non-member** functions of NDTScanMatcher?